gradle.properties

```diff
@@ -3,9 +3,9 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://modmuss50.me/fabric.html
-	minecraft_version=1.21.1
-	yarn_mappings=1.21.1+build.1
-	loader_version=0.15.11
+	minecraft_version=1.21.4
+	yarn_mappings=1.21.4+build.4
+	loader_version=0.16.9
 
 # Mod Properties
 	mod_version = 1.0.2+1.21.1
```

Comment:
You need to update the mod version to include the supported game version.

Reply:
Yeah, the mod version needs updating. I usually leave that to the original devs when I make pull requests, as I'm never certain of what versioning scheme they go with.

Because while most use schematic versioning. What is considered a major, minor, or patch release seems to vary person to person. So I just leave it for them to decide when they merge and release.

Reply:
All good :+1: